### ISSUE TITLE
razvanbrb: Testing, week 1

### ISSUE THREAD

**Issue by razvanbrb** [HackYourFutureBelgium/class-9-10#455]
# Wednesday Check-In

__Testing, week 1__

## Progress

What have you learned and/or completed so far this week?
I have isolate exercise and  integrate testing and now I'm busy with document-and-pass

## Blocked

What are you having trouble understanding? What parts of the homework are you struggling to complete?

I did not understood exactly what do we have to in Codewars.  I know that you asked if there are any questions but it was past 4:00 PM and maybe you. Sometimes I as a student  don't know what to ask because we don't know how to do the exercise to ask until we start the exercise.

## Next Steps

What are your goals and your study plans for the rest of the week?

Finish the exercise document-and-pass and Codewars and the the project for this week.

## Tip(s) of the week

Share some tips, tricks or links that made a big difference for you.

It is possible when you explain the homework for the week to do also an example after explaining it? Or because of the lack of time you can not show us how to do it?  I don't know if you understand me. I understand better if you explain and after that also show us one exercise how it has to be done.

I found testing very necessary for a developer not only for a tester.

**Comment by colevandersWands** [comment]
@razvanbrb.  We were running late on time Sunday and you're right that it's tricky to explain/understand the exercises before you've tried them (especially the Behavior, Strategy, Implementation)

Come by the review call today and we can go over how to do the exercises

**Comment by razvanbrb** [comment]
@colevandersWands  Yes your right. It's hard for you to explain everything in short time and give also example and answer all of the question. Time flyes so fast. P.S Unfortunatly I could not come by review yesterday. I will watch the recording. Thank you Evan!

**Comment by colevandersWands** [comment]
@razvanbrb, I'll be keeping an eye on the project board.  move your card to blocked if you have any questions

best of luck!
